src/main/resources/com/openshift/jenkins/plugins/pipeline/OpenShiftGlobalVariable/help.jelly

```diff
@@ -53,13 +53,13 @@
                     Declares that OpenShift operations within the closure body should be executed against the
                     specified clusterName. The statement also implicitly acts as both a
                     <a href="#openshift_withProject"><code>openshift.withProject</code></a> (establishing the default project within
-                    the closure) and <a href="#openshift_doAs"><code>openshift.doAs</code></a> (establishing the default authorization
+                    the closure) and <a href="#openshift_withCredentials"><code>openshift.withCredentials</code></a> (establishing the default authorization
```

Comment:
again, a short blurb in here where it makes sense to cite that doAs is deprecated (still works, but will be removed at some point)

Reply:
updated in next push